test/e2e/common_test.go

```diff
@@ -629,23 +631,33 @@ func isRootless() bool {
 }
 
 func SkipIfCgroupV1(reason string) {
-	if len(reason) < 5 {
-		panic("SkipIfCgroupV1 must specify a reason to skip")
-	}
+	checkReason(reason)
 	if !CGROUPSV2 {
 		Skip(reason)
 	}
 }
 
 func SkipIfCgroupV2(reason string) {
-	if len(reason) < 5 {
-		panic("SkipIfCgroupV2 must specify a reason to skip")
-	}
+	checkReason(reason)
 	if CGROUPSV2 {
 		Skip(reason)
 	}
 }
 
+func isContainerized() bool {
+	if os.Getenv("container") == "podman" {
```

Comment:
Is this the only value of "container" for which the test calling this function should be skipped?

Reply:
Well we are running within Podman, but making this just check `if os.Getenv('container") != "" ` is probably better.

Reply:
I believe so.  Though perhaps a `os.Getenv("container") != ""` would be safer.  I'll fix it, thanks.